.github/workflows/test.yaml

```
@@ -0,0 +1,19 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+on: push
+name: Test
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Install
+      run: npm install
```

Obviously both do very similar things but, among others reasons, I saw a difference of 5s in run time. I assume `ci` does not run the update check.

```suggestion
      run: npm ci
```